colin-api/src/colin_api/models/filing.py

```diff
@@ -489,7 +525,8 @@ def _insert_filing(cls, cursor, filing, ar_date: str, agm_date: str):  # pylint:
                                       'AMLHB', 'AMALH', 'AMLHU', 'AMLHC',
                                       'AMLVB', 'AMALV', 'AMLVU', 'AMLVC',
                                       'CONTB', 'CONTI', 'CONTU', 'CONTC',
-                                      'NOABE', 'NOALE', 'NOALR', 'NOALD', 'NOALA', 'NOALB', 'NOALU', 'NOALC'
+                                      'NOABE', 'NOALE', 'NOALR', 'NOALD', 'NOALA', 'NOALB', 'NOALU', 'NOALC',
+                                      'CONTO',
```

Comment:
Might look better on 2 lines.

Reply:
kept it like that (its all specific to alteration)